libexec/pyenv-init

```diff
@@ -97,10 +97,7 @@ esac
 
 completion="${root}/completions/pyenv.${shell}"
 if [ -r "$completion" ]; then
-  case "$shell" in
-  fish ) echo ". '$completion'" ;;
-  *    ) echo "source '$completion'" ;;
-  esac
+  echo "source '$completion'"
```

Comment:
Yes, seems to be good - but it confused me a bit.
Probably `source` was not available / preferred in older fish versions?!